CONTRIBUTING.md

```diff
@@ -28,7 +28,7 @@ Payment for contributions is contingent on your pull request being approved and
 ## Requirements for contributions
 
 ### Adding gems
-If you are adding a new gem, be sure to place it in the proper section of the `GEMFILE` in alphabetical order.
+If you are adding a new gem, be sure to place it in the proper section of the `GEMFILE` in no particular order.
```

Comment:
I'm very particular about this actually...

Reply:
If this isn't a problem in Beacon in particular, I'd like to keep alphabetical as the default.

Reply:
It's hard to tell if it's a problem in Beacon. How do you tell if a library is *not* doing something subtle?

I'll remove it, but I'm glad the warning got across.